device_calendar/lib/src/models/event.dart

```diff
@@ -31,6 +31,9 @@ class Event {
   /// The location of this event
   String location;
 
+  /// An URL for this event
+  String url;
```

Comment:
Update this to use the Uri class so that we can validate that the url entered conforms to a url (this is already done by iOS as it requires a Url but Android exposes this as a string)